cpp/arcticdb/storage/library_manager.hpp

```diff
@@ -38,6 +38,14 @@ namespace arcticdb::storage {
 
         [[nodiscard]] bool has_library(const LibraryPath& path) const;
 
+        [[nodiscard]] static inline bool rocksdb_support() {
```

Comment:
We can just hang thing off the module like here https://github.com/man-group/ArcticDB/blob/263e843015f010b4fcd30acb47b05a600b3d7448/cpp/arcticdb/storage/python_bindings.cpp#L38 can't we, no need for this to be tied to the `library_manager`?